pandas/tests/extension/base/getitem.py

```diff
@@ -266,3 +266,9 @@ def test_loc_len1(self, data):
         df = pd.DataFrame({"A": data})
         res = df.loc[[0], "A"]
         assert res._data._block.ndim == 1
+
+    def test_item(self, data):
+        # https://github.com/pandas-dev/pandas/pull/30175
+        s = pd.Series(data)
+        result = s[:1].item()
+        assert result == data[0]
```

Comment:
I wanted to add a more exact test for the type of the object (`assert isinstance(result, data.dtype.type)`), but this actually doesn't hold for all our dtypes (eg for categorical dtype)

Reply:
I think we had an issue about that somewhere. But this seems sufficient.